src/Assertion/Factory.php

```diff
@@ -50,12 +31,9 @@ public function create(array $specs): AssertionInterface
         return $assertionManager->get($type, $specs['options'] ?? null);
```

Comment:
```suggestion
        return $this->getAssertionPluginManager()->get($type, $specs['options'] ?? null);
```
and then you can remove the previous line